From efddab9c02e7eae1e8bae843fe4d95718478de1b Mon Sep 17 00:00:00 2001
From: Antonio Ospite <ospite@studenti.unina.it>
Date: Thu, 14 Mar 2013 23:04:20 +0100
Subject: [PATCH] doc: add a man page for am7xxx_mode_switch
X-Face: z*RaLf`X<@C75u6Ig9}{oW$H;1_\2t5)({*|jhM<pyWR#k60!#=#>/Vb;]yA5<GWI5`6u&+
 ;6b'@y|8w"wB;4/e!7wYYrcqdJFY,~%Gk_4]cq$Ei/7<j&N3ah(m`ku?pX.&+~:_/wC~dwn^)MizBG
 !pE^+iDQQ1yC6^,)YDKkxDd!T>\I~93>J<_`<4)A{':UrE

---
 doc/man/CMakeLists.txt           |    2 ++
 doc/man/am7xxx_mode_switch.1.txt |   57 ++++++++++++++++++++++++++++++++++++++
 2 files changed, 59 insertions(+)
 create mode 100644 doc/man/am7xxx_mode_switch.1.txt

diff --git a/doc/man/CMakeLists.txt b/doc/man/CMakeLists.txt
index ed46999..a21e82c 100644
--- a/doc/man/CMakeLists.txt
+++ b/doc/man/CMakeLists.txt
@@ -3,6 +3,7 @@ find_package(Asciidoc)
 if(ASCIIDOC_FOUND)
   add_custom_target(manpages
     ${ASCIIDOC_A2X_EXECUTABLE} -f manpage ${CMAKE_CURRENT_SOURCE_DIR}/am7xxx-play.1.txt -D ${DOC_OUTPUT_PATH}/man
+    COMMAND ${ASCIIDOC_A2X_EXECUTABLE} -f manpage ${CMAKE_CURRENT_SOURCE_DIR}/am7xxx_mode_switch.1.txt -D ${DOC_OUTPUT_PATH}/man
     COMMAND ${ASCIIDOC_A2X_EXECUTABLE} -f manpage ${CMAKE_CURRENT_SOURCE_DIR}/picoproj.1.txt -D ${DOC_OUTPUT_PATH}/man
     WORKING_DIRECTORY ${DOC_OUTPUT_PATH}/man
     COMMENT "Generating man pages with Asciidoc" VERBATIM
@@ -17,6 +18,7 @@ if(ASCIIDOC_FOUND)
 
   install(FILES
     ${DOC_OUTPUT_PATH}/man/am7xxx-play.1
+    ${DOC_OUTPUT_PATH}/man/am7xxx_mode_switch.1
     ${DOC_OUTPUT_PATH}/man/picoproj.1
     DESTINATION "${CMAKE_INSTALL_PREFIX}/share/man/man1/"
     COMPONENT manpages)
diff --git a/doc/man/am7xxx_mode_switch.1.txt b/doc/man/am7xxx_mode_switch.1.txt
new file mode 100644
index 0000000..d80cb60
--- /dev/null
+++ b/doc/man/am7xxx_mode_switch.1.txt
@@ -0,0 +1,57 @@
+AM7XXX_MODE_SWITCH(1)
+=====================
+:doctype: manpage
+
+
+NAME
+----
+am7xxx_mode_switch - change the operational mode of am7xxx based devices
+
+
+SYNOPSIS
+--------
+*am7xxx_mode_switch*
+
+
+DESCRIPTION
+-----------
+am7xxx_mode_switch(1) is a minimal replacement of usb-modeswitch to use with
+am7xxx devices (e.g. Acer C110 or Philips PPX projectors) to switch from the
+mass storage device mode to the generic display mode.
+
+It is handy on systems where usb-modeswitch is not available, like Windows.
+
+
+EXAMPLE OF USE
+--------------
+
+am7xxx_mode_switch
+
+
+EXIT STATUS
+-----------
+*0*::
+    Success
+
+*!0*::
+    Failure (libusb error)
+
+
+AUTHORS
+-------
+Antonio Ospite
+
+
+RESOURCES
+---------
+Main web site: <http://git.ao2.it/libam7xxx.git>
+
+
+COPYING
+-------
+Copyright \(C) 2012  Antonio Ospite <ospite@studenti.unina.it>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
-- 
1.7.10.4

